installer/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ application {
 
 dependencies {
 	// we need this older version since newer versions are Kotlin-implemented and don't play nice with GraalVM
+	// okhttp 5 will add official GraalVM support but is currently still alpha https://square.github.io/okhttp/changelogs/changelog/
 	implementation("com.squareup.okhttp3:okhttp:3.14.2")
```

Comment:
You can also add this to the `libs.versions.toml` and add add the following to the `renovate.json`:
```
    {
      "matchPackageNames": [  "com.squareup.okhttp3:okhttp" ],
      "allowedVersions": ">= 5"
    },
```
Then we'll get a PR once 5.0 is available and we cannot miss it :)

Reply:
hm, not sure this is so easy since the agent already uses a 4.* version of okhttp. So this would mean downgrading the agent's library, right? Which probably means changing a bunch of code since 3 and 4 are not API compatible, 4 being a complete rewrite in Kotlin.

Reply:
Ah sorry, I missed that we already use v4.